migrations/sql/V2019.02.21.9.59__add_address_columns_to_party.sql

```diff
@@ -0,0 +1,54 @@
+ALTER TABLE party ADD COLUMN suite_no       varchar(5);
+ALTER TABLE party ADD COLUMN address_line_1 varchar(50);
```

Comment:
I think supporting multiple addresses for a party in the future was discussed. If we can **confirm** that, it could be an opportunity to put this in a related table and avoid some future refactoring, even if today we only ever treat it as a 1:1 entity. If we can't confirm this 100% then it does make sense not to pay that cost today. Not a blocking comment.

Reply:
This was discussed. I asked Kristin if we should do it in a separate address table to support multiple, and the consensus was to put the columns directly on the party table at this time.

Reply:
Sounds good. My intuition leans the other way, but I also don't think we'll have any complexities refactoring this if needed.